packages/react/package.json

```diff
@@ -0,0 +1,93 @@
+{
+    "name": "@solana/react",
+    "version": "2.0.0-preview.2",
```

Comment:
Shall we align with `preview.3` on the other packages?

Reply:
TY!